Gemfile

```diff
@@ -213,3 +213,7 @@ gem "opentelemetry-instrumentation-action_pack"
 gem "opentelemetry-instrumentation-action_view"
 
 gem "maintenance_tasks", "~> 2.3"
+
+group :development do
```

Comment:
there is already a `group :development do` here: https://github.com/CircuitVerse/CircuitVerse/blob/0762999968fc96c632edd6a3fe5beca776b8c416/Gemfile#L154
you could have added the gem there